### ISSUE TITLE
Generalize a ProtractorNode

### ISSUE THREAD

**Issue by samreid** [phetsims/scenery-phet#181]
I noticed Pendulum Lab and Bending Light both have ProtractorNode.js.  Should we compare/contrast/generalize?  What other sims will need protractors?

**Comment by veillette** [comment]
The Protracor Node in Bending Light is an image whereas Pendulum Lab generates a protractor using scenery.

**Comment by samreid** [comment]
Is this the protractor in Pendulum Lab?

![image](https://cloud.githubusercontent.com/assets/679486/16634030/44e0b70a-4389-11e6-9d91-8180f2853908.png)

**Comment by veillette** [comment]
Yes, and the protractorNode in pendulum lab has two blue ticks (not currently displayed in your screenshot)

**Comment by pixelzoom** [comment]
Things appear to have evolved/diverged since this issue was created.

The 2 implementations in questions are:

* pendulum-lab.PendulumLabProtractorNode - scenery based, used in pendulum-lab
* scenery-phet.ProtractorNode - image based, moved from bending-light to scenery-phet on 6/17/20 for https://github.com/phetsims/scenery-phet/issues/609, used in bending-light and xray-diffration

Questions:

* Can pendulum-lab use scenery-phet.ProtractorNode ?
* Should scenery-phet.ProtractorNode be converted to a scenery-based (instead of image-based) implementation?

My feeling is that the developer of the next sim that needs a protractor should address this issue.
